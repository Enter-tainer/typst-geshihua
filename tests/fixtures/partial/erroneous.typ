#let f(x,y:)={
  let a=(1,2,3)
  let b=(,)
}

#

#let x=(:)
