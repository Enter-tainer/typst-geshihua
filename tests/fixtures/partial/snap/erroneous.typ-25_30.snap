---
source: tests/src/partial.rs
info:
  range:
    start: 25
    end: 30
  range_node:
    start: 23
    end: 30
input_file: tests/fixtures/partial/erroneous.typ
snapshot_kind: text
---
(1,2,3)
---
(1, 2, 3)
